src/sentry/integrations/msteams/card_builder/base.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import annotations
+
+from abc import ABC
+from typing import Any, Sequence
+
+from sentry.integrations.msteams.card_builder import Action, AdaptiveCard, Block
+from sentry.integrations.notifications import AbstractMessageBuilder
+
+from .block import create_text_block
+
+
+class MSTeamsMessageBuilder(AbstractMessageBuilder, ABC):  # type: ignore
```

Comment:
@97varun if we are going to use the class directly, then it shouldn't be an abstract class

Reply:
Oh no! why did I miss this, I thought inheriting form ABC should prevent me from initializing this class, let me see how I can structure this better...

Reply:
Udpated.